source-code-flow-manager-application/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM openjdk:17-jdk-alpine
+
+COPY /build/libs/source-code-flow-manager-application-*.jar /opt/server/
+
+VOLUME /opt/server/
```

This would be better. Server is too much abstract
```suggestion
VOLUME /opt/source-code-flow-manager-app/
```

---

It's fixed.